app/views/IncomeTaxSummaryView.scala.html

```diff
@@ -45,26 +45,39 @@
 
 @header = @{
     ptaHeader(
-        displayBackLink = true,
+        displayBackLink = false,
```

Comment:
should this be false, do we not need a backlink?

Reply:
no backlink is now handled by the new main template however we still need in the above header for older existing pages